media/js/app/assetmgr/collection.js

```diff
@@ -482,7 +482,7 @@ CollectionList.prototype.updateSwitcher = function () {
     Mustache.update("switcher_collection_chooser", self.switcher_context, { parent: self.parent });
 
     // hook up switcher choice owner behavior
-    jQuery(self.el).find("a.switcher-choice.owner").unbind('click').click(function (evt) {
+    jQuery(self.el).find("a.switcher-choice.owner").off('click').on('click', function (evt) {
```

Comment:
Thanks for experimenting. When we have some time, it might be interesting to dig deeper into this. But I'm good going with this now.